packages/remark-custom-blocks/src/index.js

```diff
@@ -1,7 +1,7 @@
 const spaceSeparated = require('space-separated-tokens')
 
 function escapeRegExp (str) {
-  return str.replace(/[-[]\/\{\}\(\)\*\+\?\.\\\^\$\|]/g, '\\$&')
+  return str.replace(new RegExp(`[-[\\]{}()*+?.\\\\^$|/]`, 'g'), "\\$&")
```

Comment:
> This would replace any match with `undefined`:
> 
> ```js
> > escapeRegExp('[AA-AA|AA]AA')
> 'undefinedAAundefinedAAundefinedAAundefinedAA'
> ```

I forgot to add the `\\&$` for the replacement string. I've updated the commit.

![image](https://user-images.githubusercontent.com/8973405/60967338-6d37da00-a2e8-11e9-89f9-ad06316793c0.png)